### DISCUSSION TITLE
audio and visual notification for vehicle faster than maxspeed

### DISCUSSION THREAD

**Discussion by rdorsch** [osmandapp/OsmAnd#12475] [General]
Hi,

I used navit before osmand and a handy feature was that it warned the driver if the vehicle has a higher speed than the maxspeed allowed. There have been even config options for the warning limits, e..g 5% faster than max speed but at least 5 km/h.

I found that osm can display speed limits, but I have not yet seen that osmand gives any notificatino that the speed limit is exceeded (or I did not yet go fast enough ;-) ).

Any advice or hint is welcome.

Thanks
Rainer

**Comment by pebogufi** [comment]
Einstellungen / Navigationseinstellungen /Sprachansagen
